docs/guide/localization/static.md

```diff
@@ -0,0 +1,199 @@
+
+### Registering text outside of templates
+
+Two other places you may want to register strings are in **API routes** and in Vue.js **user interface components**.
+
+To register strings in **custom API routes**, use the `req.t()` method on the request object.
+
+<AposCodeBlock>
+  ```javascript
+  module.exports = {
+    // ...
+    apiRoutes(self) {
+      return {
+        get: {
+          // GET /api/v1/product/newest
+          async newest(req) {
+            const product = await self.find(req).sort({ createdAt: -1 }).toObject();
+            if (!product) {
+              // 👇 The 404 error is registered with req.t()
+              throw self.apos.error('notfound', req.t('No products were found.'));
+            }
+
+            return { product };
+          }
+        }
+      };
+    }
+  };
+  ```
+  <template v-slot:caption>
+    /modules/product/index.js
+  </template>
+</AposCodeBlock>
+
+::: tip
+There is currently no browser method available to register strings in project client-side JS. A good alternative is to create an API route that returns localized strings then use the response from that route in client-side JS.
+:::
+
+If you are customizing Apostrophe **user interface components**, the `$t()` method is available in the component (used as `this.$t()`) outside the template block.
+
+<!-- TODO: Link to UI customization guide when available. -->
+
+::: note
+As a reminder, the Vue.js components of the user interface are not connected to any Vue app you may be running for your website visitors. The registration method will not be automatically available outside the UI components.
+:::
+
+## Adding and using localization files
+
+Now that there are strings to localize, we need to add JSON files for locales with the strings and their translations. For the purposes of our example we will assume we have two locales: `'en'` (English) and `'es'` (Spanish).
+
+Each locale should get its own JSON file using the locale name, in a `i18n` directory of a module that has internationalization active. Configure that module with the [`i18n` option](/reference/module-api/module-options.md#i18n), such as:
+
+<AposCodeBlock>
+  ```javascript
+  module.exports = {
+    options: {
+      i18n: true
+    }
+  };
+  ```
+  <template v-slot:caption>
+    /modules/localization/index.js
+  </template>
+</AposCodeBlock>
+
+Like all modules, we would need to activate this in `app.js`. Our JSON files would be:
+
+- `modules/localization/i18n/en.json`
+- `modules/localization/i18n/es.json`
+
+::: tip
+These files can be spread across modules if you like. For example you might want to localize strings from a particular module's template files in that module directory. Like with many things, this depends how you and your team like to work.
+
+It's important to note that **if the same key is used in multiple template files they will be rendered to the same string**. It does not matter if each template has a localization JSON file with that key registered.
```

Comment:
People really want to know where to put stuff, and they will put it in `@apostrophecms/i18n` because that sounds right. But they should not put their own phrases there, unless it's for the apostrophe admin UI, because that module is already using the `apostrophe` namespace. So we should explicitly note that you should not put your project level phrases in that module at project level.